src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -187,6 +187,10 @@ protected void LookupMembersInType(LookupResult result, TypeSymbol type, string
                     this.LookupMembersInClass(result, type, name, arity, basesBeingResolved, options, originalBinder, diagnose, ref useSiteDiagnostics);
                     break;
 
+                case TypeKind.Tuple:
+                    useSiteDiagnostics = LookupMembersInTuple(result, type, name, arity, basesBeingResolved, options, originalBinder, diagnose, useSiteDiagnostics);
+                    break;
```

Comment:
Why not use `ref useSiteDiagnostics` like neighboring code? It would be more consistent.